logos/benchmarks/benchmark_logos.py

```
@@ -1248,6 +1284,386 @@ def generate_charts(out_dir: Path, results: list[RequestResult], tracker, t0: fl
+    use_sudo: bool,
+) -> None:
+    """Start the logos workernode on each GPU node via SSH docker compose up -d."""
+    sudo = "sudo " if use_sudo else ""
+    remote_cmd = f"cd {shlex.quote(workernode_dir)} && {sudo}docker compose up -d"
+    for host in hosts:
+        parts = ["ssh", "-o", "StrictHostKeyChecking=no"]
+        if ssh_key:
+            parts += ["-i", ssh_key]
+        parts += [f"{ssh_user}@{host}", remote_cmd]
+        print(f"  [logos] {host}: $ {remote_cmd}")
+        result = subprocess.run(parts)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)